lib/routes/destroy.js

```diff
@@ -5,29 +5,38 @@
 const Joi = require('joi');
 
 const AppError = require('../error');
-const config = require('../config');
 const db = require('../db');
 const encrypt = require('../encrypt');
 const validators = require('../validators');
 
+/*jshint camelcase: false*/
+
 module.exports = {
   validate: {
-    payload: {
-      token: Joi.string()
-        .length(config.get('unique.token') * 2) // hex = bytes*2
-        .regex(validators.HEX_STRING)
-        .required()
-    }
+    payload: Joi.object().keys({
+      access_token: validators.token,
+      refresh_token: validators.token
+    }).rename('token', 'access_token').xor('access_token', 'refresh_token')
```

Comment:
Seems obvious to me, since token is now ambiguous, more specific names were needed.